fastlane/Fastfile

```diff
@@ -638,16 +689,23 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Builds an app bundle"
   lane :build_for_translation_review do | options |
+
+  translator_locales = %w[de es fr id it ja ko nl ru sv zh-cn zh-tw tr he pt ar pl th sr vi]
+  translator_locales_map = ALL_LOCALES.select { translator_locales.include?(h[:glotpress]) }
```

Comment:
Indentation fix
```suggestion
    translator_locales = %w[de es fr id it ja ko nl ru sv zh-cn zh-tw tr he pt ar pl th sr vi]
    translator_locales_map = ALL_LOCALES.select { translator_locales.include?(h[:glotpress]) }
```

Reply:
Oh and also we forgot to provide the param name to the `select` block (meaning that current code would crash with syntax error)
```suggestion
    translator_locales = %w[de es fr id it ja ko nl ru sv zh-cn zh-tw tr he pt ar pl th sr vi]
    translator_locales_map = ALL_LOCALES.select { |h| translator_locales.include?(h[:glotpress]) }
```